app/src/main/java/com/eveningoutpost/dexdrip/utilitymodels/NightscoutBatteryDevice.java

```diff
@@ -143,6 +145,9 @@ boolean alwaysSendBattery() {
         @Override
         String getDeviceName() {
             if (Ob1G5StateMachine.usingG6()) {
+                if (shortTxId()) { // If using G7
+                    return "G7 Device";
```

Comment:
change to "G7 Transmitter";

Reply:
G7 transmitter and sensor are together.  There is no reason to call it a transmitter any longer.  It is valid to call it a device to avoid calling it a transmitter.